Scarb.toml

```
@@ -39,8 +39,9 @@ keywords = [
 ]
 
 [workspace.dependencies]
-starknet = "2.8.0"
-snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.27.0" }
+assert_macros = "2.8.2"
+starknet = "2.8.2"
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.30.0" }
```

Do you think it's time we declare this from the registry? Not sure if there are any performance changes
```suggestion
snforge_std = "0.30.0"
```

---

Good call! I haven't seen any benchmarks on performance, but I saw tat as part of Scarb 2.8.2 some bug that caused Unnecessary registry downloads was fixed, so we should be good.